### PR TITLE
Update CC & Tutor QL help text

### PR DESCRIPTION
**Pull request by nathanstitt**
Tutor top bar  (spacing is shown as-is directly after section choice):
screen shot 2016-07-26 at 4 16 06 pm

Tutor second bar:
screen shot 2016-07-26 at 4 16 34 pm

CC top bar:
screen shot 2016-07-26 at 4 16 45 pm

CC bottom bar:
screen shot 2016-07-26 at 4 17 52 pm

### CODE REVIEW

**Comment by nathanstitt** [comment]
@kjdav please take a look at this to make sure wording is correct.  Thanks!

**Comment by kjdav** [comment]
Wording looks good!

**Comment by pandafulmanda** [comment]
minor, tooltip pops on top of nav bar: 
screen shot 2016-07-27 at 4 02 30 pm 1

File: src/components/questions/exercises-display.cjsx [hunk 1]
```diff
@@ -216,6 +211,12 @@ ExercisesDisplay = React.createClass
         cardType='sections-questions'
       >
 
+      <div className="instructions">
+        <div className="wrapper">
+          {Help.forCourseId(@props.courseId).second.bar}
```
Comment (pandafulmanda) [inline src/components/questions/exercises-display.cjsx]: This `Help` thing is pretty kewlz.

**Pull request merged**